src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2119,8 +2119,9 @@ internal enum ErrorCode
         #endregion
 
         #region diagnostics introduced for C# 12.0
-        ERR_ImplicitlyTypedDefaultParameter = 9066,
-        ERR_OptionalParamValueMismatch = 9067,
+        ERR_ImplicitlyTypedDefaultParameter = 9067,
```

Comment:
While we're here, consider moving these farther out, like 9500 or so, and leaving a prototype comment to condense before merge so we don't run into this again until we're ready to merge back to main.